### ISSUE TITLE
Genesis help does not know about "add" or "use"

### ISSUE THREAD

**Issue by krutten** [starkandwayne/genesis#71]
Genesis "help" is missing some of the cammands needed for normal operations like how to add a release

```
$ genesis help
USAGE: genesis <command> [arguments]

Common commands:

    genesis new deployment      Create a new deployment directory, with all the
                                necessary files for deploying with Genesis.

    genesis new site            Set up the files and directories for a new site.

    genesis new environment     Set up the files and directories for a new environment.

    genesis build               When run from inside of an environment directory,
                                paste together all of the template files, in order
                                and pop out a manifest file.

    genesis diff                When run from inside of an environment directory,
                                show a semantic diff of the generated manifest for that
                                environment, against some other manifest file.

    genesis refresh             Refresh the cached copies of site and/or global
                                definitions inside the current environment directory.
                                Can also be used to recreate the Makefile after a
                                Genesis upgrade.

    genesis bosh                Generates a deployable manifest (which may have secrets
                                in it) and then runs the `bosh` CLI with remaining
                                arguments.  Useful for recreating workers.

    genesis help                Peruse the help!  Give it a topic argument, or the name
                                of another command to get more detailed information.
                                Documentation!

$ genesis help add
Unrecognized help topic 'add'.
Try one of these:

    genesis help usage
    genesis help <command>

$ genesis add release rabbitmq
Using the latest version of rabbitmq available via the Genesis Index
$
```

We should make sure known commands show up in at least the initial help page

**Comment by jhunt** [comment]
Partially this was undocumented because we weren't sure we liked the `add` / `use` / `set` verbs, esp. since their usage is distinct, but their distinctions are not so much (particularly, `use` vs. `set`)

**Comment by jhunt** [comment]
... and once you document something, people start to use it.  😄

**Comment by jhunt** [comment]
Fixed in #74
